cms/static/sass/views/_group-configuration.scss

```diff
@@ -59,12 +59,12 @@
           @extend %cont-truncated;
           @extend %t-title5;
           @extend %t-strong;
-          margin-right: ($baseline*14);
+          @include margin-right($baseline*14);
           color: $black;
 
           .toggle {
+            @include padding-left($baseline);
```

Comment:
Actually, we can take the padding left out. If this is the only place that has the toggle class.
<img width="1178" alt="screen shot 2016-11-03 at 9 43 58 am" src="https://cloud.githubusercontent.com/assets/16127998/19968245/2d5aa0d0-a1aa-11e6-8a75-ab368a161ed0.png">